Content.Server/Guardian/GuardianSystem.cs

```diff
@@ -195,7 +205,7 @@ private void OnCreatorInject(GuardianCreatorInjectedEvent ev)
                 guardianComponent.Host = ev.Target;
 
                 // Grant the user the recall action and notify them
-                actions.Grant(ActionType.ManifestGuardian);
+                //actions.Grant(ActionType.ManifestGuardian);
```

Comment:
?

Reply:
When initially stripping out old actions I just commented out all the code that was adding actions, and just forgot to remove this. Will remove, along with some unnecessary action TryGet & Ensures in the guardian system